wallet-connect-test-bench/front-end/package.json

```diff
@@ -52,7 +50,7 @@
         "typescript": "^4.7.2"
     },
     "scripts": {
-        "preinstall": "cd ../../deps/concordium-dapp-libraries && yarn install && yarn build",
+        "preinstall": "cp -r ../../deps/concordium-dapp-libraries/packages/react-components ./node_modules/@concordium/react-components; cp -r ../../deps/concordium-dapp-libraries/packages/wallet-connectors ./node_modules/@concordium/wallet-connectors",
```

Comment:
I'm not sure about reverting these changes.

I had no problems with the setup that Soren did and it looks to be the intended way as well.

What was the issue exactly?

Reply:
Can @soerenbf pitch in?

Reply:
So the issue I had when first cloning the repo and tried installing for the first time, the `node_modules` folder was not present, but required by the `"preinstall"` node script. I'm sure it would have worked if I added the folder, but thought I might as well clean it up and align with best practice (as far as I'm aware at least).

What I did not think of however, was how this would work with the `Dockerfile` build, which might be the reason for reverting (@DOBEN?).

Might be able to improve this even more by referencing the TS dependencies from git submodules through the ts-config instead as well, which would omit the requirement of building the dependencies before being able to use them.

Reply:
It was resolved in a separate PR by keeping local dependencies:
https://github.com/Concordium/concordium-misc-tools/pull/66